scenarioo-server/src/main/java/org/scenarioo/dao/search/IgnoreUseCaseSetStatusMixIn.java

```diff
@@ -0,0 +1,26 @@
+/* scenarioo-server
+ * Copyright (C) 2014, scenarioo.org Development Team
+ * 
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.scenarioo.dao.search;
+
+import org.codehaus.jackson.annotate.JsonIgnore;
+import org.scenarioo.model.docu.entities.Status;
+
+public abstract class IgnoreUseCaseSetStatusMixIn {
+	@JsonIgnore
+	public abstract void setStatus(Status value);
```

Comment:
@featureenvy: I still not get it. Why should this setter ever be used?? this field is a pure String field. This special setter with the Status enum as a parameter will never be used when deserializing in my oppinion, it is simply a utility method for convenience, that is not relevant ever when deserializing, because it will use setStatus(String) instead. (sorry for only seeing your comment on that now). Hmm, maybe we have to discuss that once.